docs/ansible/role-index.rst

```diff
@@ -160,9 +160,6 @@ Logging
 - :ref:`debops.kibana`
 - :ref:`debops.logrotate`
 - :ref:`debops.rsyslog`
-- :ref:`debops.prometheus_server`
-- :ref:`debops.prometheus_exporters`
-- :ref:`debops.prometheus_alertmanager`
```

Comment:
You can definitely put roles in multiple index sections, it will be easier to find them that way.

Reply:
I don't think it matches logging, it was a misstage to put it there.
It matches much more monitoring.